presto-main/src/main/java/com/facebook/presto/memory/ClusterMemoryManager.java

```diff
@@ -240,14 +244,22 @@ public synchronized void process(Iterable<QueryExecution> runningQueries, Suppli
             return;
         }
 
-        // TODO revocable memory reservations can also leak and may need to be detected in the future
-        // We are only concerned about the leaks in general pool.
+        Map<QueryId, Optional<BasicQueryInfo>> localOrGloballyRunningQueries;
```

Comment:
let's just call `globalRunningQueries` or `runningQueries` as it represents same either in single coordinator/multi coordinator.